plugin/pkg/shoot/validator/admission.go

```diff
@@ -508,11 +508,18 @@ func validateKubernetesVersionConstraints(constraints []core.ExpirableVersion, s
 	validValues := []string{}
 	var latestVersion *semver.Version
 	for _, versionConstraint := range constraints {
-		if versionConstraint.ExpirationDate != nil && versionConstraint.ExpirationDate.Time.Before(time.Now()) {
+		if versionConstraint.ExpirationDate != nil && versionConstraint.ExpirationDate.Time.UTC().Before(time.Now().UTC()) {
 			continue
```

Comment:
Can we return an error, if the Shoot's used k8s version is expired? Or are we already doing that elsewhere?

Reply:
I see what your are saying, though I think that is not related to this PR. It would require some rework in the shoot admission. Though it is not a major effort, I would like to have that in a dedicated  PR.

Reply:
Fine for me 👍 Can you open an issue to track that?

Reply:
Please see [this issue.](https://github.com/gardener/gardener/issues/2136)